car/app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+  has_many :car_models
+  has_many :type_of_cars
+end
```

Comment:
think you may have wanted a has_many ... , through here. is this for a car collection or just a classification? if it's just a classification for a car, it can be probably just be a property/column in CarModel